src/prototype_room_my.js

```diff
@@ -284,7 +284,31 @@ Room.prototype.getHarvesterAmount = function() {
   return amount;
 };
 
+Room.prototype.visualizz = function() {
+
+  let color = function(coeff) {
+    return '#' + (Math.floor(255 - coeff * 239)).toString(16) + (Math.floor(coeff * 239 + 16)).toString(16) + '00'
+  }
+  this.visual.text(`Energie Average : ${Math.floor(this.memory.energyAvailable)}`, 5,0.5, {
```

Comment:
Change `Energie Average` to `Available Energy`

Reply:
it's is the average energy used by helpers. it's just badly named.